src/ex4/server/services/item_manager.js

```diff
@@ -1,86 +1,103 @@
 // The ItemManager should go here. Remember that you have to export it.
-import { promises as fs } from "fs";
-import { PokemonClient } from '../clients/pokemon_client.js';
-const tasksFile = 'tasks_list.json';
+const fs = require("fs/promises");
+const PokemonClient = require('../clients/pokemon_client.js');
+const { Todo } = require('../db/models');
+
+module.exports = class ItemService {
 
-export class ItemService {
     async addTask(task) {
         const response = [];
-        let data = await this.readTasksFile();
-        if (!data) data = [];
-
-        if (this.isListOfNumbers(task.name)) {
-            const idsList = task.name.split(',');
+        if (this.isListOfNumbers(task.taskName)) {
+            const idsList = task.taskName.split(',');
             const pokemons = await PokemonClient.getPokemons(idsList);
-            let serialNum = task.serialNum;
+
             let i = 0;
             pokemons.forEach(pokemon => {
-                response.push({ name: pokemon, color: task.color, urgency: task.urgency, serialNum: serialNum, pokemonId: idsList[i] });
-                serialNum++;
+                response.push({
+                    serialNum: task.serialNum, taskName: pokemon,
+                    urgency: task.urgency, pokemonId: idsList[i],
+                    status: task.status
+                });
                 i++;
+                task.serialNum++;
             });
-            data = [...data, ...response];
-            
         } else {
             response.push(task);
-            data = [...data, ...response];
         }
-        await this.writeTasksFile(data);
+        response.forEach(async task => { await this.createTodo(task) });
         return response;
     }
 
+    async createTodo(todo) {
+        await Todo.create({
+            "serialNum": todo.serialNum,
+            "taskName": todo.taskName,
+            "urgency": todo.urgency,
+            "pokemonId": todo.pokemonId,
+            "status": todo.status
+        });
+    };
+
     isListOfNumbers(str) {
         const arr = str.split(',');
         if (arr.findIndex(item => { return item === '' }) >= 0) return false;
         return !arr.find(item => (isNaN(item.trim())));
     }
 
     async getAll() {
-        return await this.readTasksFile();
+        const data = await Todo.findAll();
+        return (data.map(todo => todo.dataValues));
     }
 
     async deleteTask(taskName) {
-        const data = await this.getAll();
-        const index = data.findIndex((value) => value.name === taskName);
-        const deletedTask = data[index]
-        data.splice(index, 1);
-        await this.writeTasksFile(data);
-        return deletedTask;
+        const deletedTask = await Todo.findOne({
```

Comment:
why do we need to return the data of the deleted task?
just do 
```suggestion
    await Todo.destroy({
      where: { taskName}
      });
```